app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -45,6 +46,24 @@ class NavigationToolbarRobot {
 
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
+        fun clearAddressBarActions(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
+            mDevice.waitNotNull(
+                Until.findObject(By.res("org.mozilla.fenix.debug:id/toolbar")),
+                waitingTime
+            )
+            urlBar().click()
```

Comment:
Not sure why you  need this...once you are in browserRobot and tap on urlBar,  you are already on navigationToolBar... 🤔

Reply:
To make the url bar into edit mode?

Reply:
Oh that was by mistake :/

Reply:
not a mistake :) you can enter edit mode  there but it  is not necessary in this case.

Reply:
Ok.